Description: Loosen syn version requirement
Author: Abhinaba Rakshit <abhinaba.rakshit@oss.qualcomm.com>
Origin: other, Debian packaging
Last-Update: 2026-04-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: mink-idl-compiler/idlc_codegen_rust/Cargo.toml
===================================================================
--- mink-idl-compiler.orig/idlc_codegen_rust/Cargo.toml
+++ mink-idl-compiler/idlc_codegen_rust/Cargo.toml
@@ -11,4 +11,4 @@ idlc_codegen = { path="../idlc_codegen"
 idlc_errors = {path = "../idlc_errors" }
 idlc_mir = { path="../idlc_mir" }
 prettyplease = "0.2.6"
-syn = { version = "2.0.60", features = ["parsing"], default-features = false }
+syn = { version = "2.0.48", features = ["parsing"], default-features = false }
